kumascript/macros/CSSSyntax.ejs

```diff
@@ -427,7 +429,8 @@ if (name === "preview-wiki-content") {
 }
 
 if (!formattedSyntax) {
-    throw new Error(`No syntax found in DB for '${name}'`);
+    // throw new Error(`No syntax found in DB for '${name}'`);
+    console.warn(`No syntax found in DB for '${name}'`);
```

Comment:
Is this just a temporary hack? If so, perhaps replace the commented-out old code with a URL to an issue that gives more context to this. 
Perhaps explain *why* this is done and why it's OK even for en-US docs to let it slip when the syntax is missing.

Reply:
👍